File: chessgame-api/src/main/java/git/dimitrikvirik/chessgameapi/controller/UserController.java
//package git.dimitrikvirik.chessgameapi.controller;
//
//import git.dimitrikvirik.chessgameapi.facade.AuthFacade;
//import git.dimitrikvirik.chessgameapi.model.param.UserRegParam;
//import lombok.RequiredArgsConstructor;
//import org.springframework.web.bind.annotation.PostMapping;
//import org.springframework.web.bind.annotation.RequestBody;
//import org.springframework.web.bind.annotation.RequestMapping;
//import org.springframework.web.bind.annotation.RestController;
//
//import javax.validation.Valid;
//
//@RestController
//@RequestMapping("/user")
//@RequiredArgsConstructor
//public class UserController {
//    private final AuthFacade authFacade;
//
//    @PostMapping
//    public void registration(@Valid @RequestBody  UserRegParam userRegParam) {
//        authFacade.registration(userRegParam);
//    }
//}

File: chessgame-api/src/main/kotlin/git/dimitrikvirik/chessgameapi/controller/UserController.kt
package git.dimitrikvirik.chessgameapi.controller

import git.dimitrikvirik.chessgameapi.model.param.UserParam
import org.springframework.boot.autoconfigure.jdbc.JdbcTemplateAutoConfiguration
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.RestController
import javax.validation.Valid

@RestController
class UserController {

    @PostMapping("/registration")
    fun registration(@Valid @RequestBody userParam: UserParam) {

    }

}